variantlib/models/variant.py

```diff
@@ -44,17 +41,68 @@ class VariantMeta(BaseModel):
             )
         }
     )
-    key: str = field(
+    feature: str = field(
         metadata={
             "validator": lambda val: validate_and(
                 [
                     lambda v: validate_instance_of(v, str),
-                    lambda v: validate_matches_re(v, VALIDATION_KEY_REGEX),
+                    lambda v: validate_matches_re(v, VALIDATION_FEATURE_REGEX),
                 ],
                 value=val,
             )
         }
     )
+
+    @property
+    def feature_hash(self) -> int:
+        return hash((self.namespace, self.feature))
+
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, VariantFeature):
+            return NotImplemented
+        return self.namespace == other.namespace and self.feature == other.feature
```

Comment:
This also looks redundant to what `dataclass` creates automatically.

Reply:
Let me try to remove this

Reply:
Fixed in 348bd2aeeaeb687ef876c7ee1d7a02bbcf7b46a2